app/schema/answer.py

```diff
@@ -58,14 +56,12 @@ def check_user_input(user_input):
 
     def get_typed_value(self, raw_input):
 
-        user_input = bleach.clean(raw_input)
```

Comment:
That might be ok for people's names, but if we ever have inputs for something like a company name blocking '<' or '>' probably isn't what we want to do e.g. MORE TH>N.